cafebabel/templates/base.html

```diff
@@ -3,6 +3,7 @@
 <meta charset=utf-8>
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
+<link rel=icon href="data:;base64,iVBORw0KGgo=">
```

Comment:
Maybe adding a comment about why that icon (for now?)

Reply:
yep ! https://github.com/cafebabel/cafebabel.com/pull/436/commits/9f5ea3dfd04e6e387ff643c9e089146c1f1db80c